### DISCUSSION TITLE
Shutter Open/Close Units

### DISCUSSION THREAD

**Discussion by Odotocodot** [mitsuba-renderer/mitsuba3#994] [Q&A] [answered]
Hi

I can't seem to find the units for shutter_open and shutter_close. Are they in seconds or milliseconds (~or years~)?

Thanks!

**Comment by wjakob** [comment, marked as answer]
They are in whatever unit you use to specify time elsewhere. The interpretation is for you to decide.

**Reply by Odotocodot** [reply]
I see 🤔.
Where would I specifiy time elsewhere? Do you mean outside of mitsuba? Currently I have a extremely simple scene with a cube, and I am tweaking the sensor `shutter_close` time, and want to know by what magnitude.

**Reply by wjakob** [reply]
We don't really have work motion blur in Mitsuba 3 at the moment. So these parameters are AFAIK ignored, unless you implemented a custom integrator that uses them.

**Reply by Odotocodot** [reply]
Ah okay. I was disillusioned by the [batch sensor test](https://github.com/mitsuba-renderer/mitsuba3/blob/master/src/sensors/tests/test_batch.py#L33), I was under the impression shutter time had been implemented at least for the path intergator.

Thanks for the clarification.
